bundle/manifests/dscinitialization.opendatahub.io_dscinitializations.yaml

```diff
@@ -47,17 +47,21 @@ spec:
             properties:
               applicationsNamespace:
                 default: opendatahub
+                description: Namespace for applications to be installed
```

Comment:
Can we just add here that, currently this is non-configurable?

Reply:
updated